.github/workflows/label.yaml

```diff
@@ -0,0 +1,82 @@
+name: Label Pull Requests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - unlabeled
+
+permissions:
+  pull-requests: write
+
+jobs:
+  external-contributor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Author Association and Label PR
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const pr = context.payload.pull_request;
+            if (!pr) {
+              console.log("Could not get PR from context");
```

Comment:
This should probably fail and indicate an error, right? Maybe we want a label like 'UNLABELABLE' or something? But i guess without the PR there is nothing to label?

Reply:
I switched to core.error which I think will add an annotation.